docs/source/openvino/optimization.mdx

```diff
@@ -1005,7 +1005,6 @@ The speech-to-text Whisper model can be quantized without the need for preparing
 model_id = "openai/whisper-tiny"
 ov_model = OVModelForSpeechSeq2Seq.from_pretrained(
     model_id,
-    stateful=False,
```

Comment:
now that the model is stateful, we should also modify the line below to remove the decoder-with-past mention

Reply:
Good catch! Thanks